.github/workflows/lint.yml

```diff
@@ -0,0 +1,21 @@
+# lint.yml
```

Comment:
Instead of adding a new workflow, can you include this in the main workflow? Github treats different workflow files as different pipelines, so it makes it harder to see everything at once.

Reply:
done